pkg/network/master/egressip.go

```diff
@@ -163,6 +164,18 @@ func (eim *egressIPManager) poll(stop chan struct{}) {
 	}
 }
 
+func nodeIsReady(node *v1.Node) bool {
+	nodeReady := true
+	for _, cond := range node.Status.Conditions {
+		if cond.Type == v1.NodeReady {
+			if cond.Status == v1.ConditionFalse || cond.Status == v1.ConditionUnknown {
+				nodeReady = false
```

Comment:
you can just `return false` here (and `return true` at the end; you don't need the `nodeReady` variable)